tdrs-backend/docs/jwt_key_management.md

```diff
@@ -0,0 +1,22 @@
+# JWT KEY Management for Local Development
+
+**Description**
+
+
+The JWT_KEY and JWT_CERT_TEST environment variables references values not permitted to be stored in any repo or documentation. Due to the need of these values for Login.gov functionality 
+ 
+Some backg
```

Comment:
This documentation looks incomplete. Since it's not strictly related to roles and permissions, we could break out documentation of JWT keys into a separate PR.

Reply:
Makes sense

Reply:
@alexsoble This has been removed from the PR

Reply:
Thanks! Though I'm still seeing "to create and update  roles" when I look at the changed files, should we remove that as well?

I'm also seeing "Accepts GET and requests".

Reply:
Oh sorry, left the above comment in the wrong thread!

Reply:
My only thought here is that we should update the tdrs-backend/README documentation that references the JWT key documentation if we remove the JWT key documentation

Reply:
Makes sense - this is done

Reply:
Hmmm, I'm still seeing it when I look at /files

Reply:
Not sure what happened there, but it's updated now

Reply:
👍 👍